src/main/java/org/candlepin/model/OwnerProductCurator.java

```diff
@@ -1119,7 +1128,8 @@ public void rebuildOwnerProductMapping(Owner owner, Map<String, String> productI
 
             Query query = this.getEntityManager()
                 .createNativeQuery(sql)
-                .setParameter("owner_id", owner.getId());
+                .setParameter("owner_id", owner.getId())
+                .setHint(QueryHints.NATIVE_SPACES, EnvironmentContent.class.getName());
```

Comment:
If I'm understanding the NATIVE_SPACES stuff correctly, this should be using the `OwnerProduct` class rather than `EnvironmentContent`.

Reply:
Good call, also, I can't find anybody calling rebuildOwnerProductMapping(). What would you think about removing the method entirely?

Reply:
keeping per separate conversation, and yes, that should be OwnerProduct instead of EnvironmentContent.